docs/noobaa.md

```diff
@@ -0,0 +1,15 @@
+# Noobaa Commands
+
+The Noobaa command supports all the noobaa cli arguments available in odf.
+
+## Example
+
+```bash
+odf noobaa version
```

Comment:
Could we run a noobaa command in the CLI to get basic validation that some command succeeds? Or maybe not if all commands require noobaa to be running.

Reply:
You mean to add in CI?

@travisn actually, we don't have any validation CI for downstream apart from test and build. We never agreed to run validatoin in github action with Kubernetes as we wanted to validate on OpenShift environment.